upgrade/upgradetest/config.go

```diff
@@ -0,0 +1,74 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package upgradetest
+
+import (
+	"time"
+
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/upgrade"
+)
+
+// GetConfig returns an upgrade config with the provided fork scheduled to have
+// been initially activated and all other forks to be unscheduled.
+func GetConfig(fork Fork) upgrade.Config {
+	return GetConfigWithUpgradeTime(fork, upgrade.InitiallyActiveTime)
+}
+
+// GetConfigWithUpgradeTime returns an upgrade config with the provided fork
+// scheduled to be activated at the provided upgradeTime and all other forks to
+// be unscheduled.
+func GetConfigWithUpgradeTime(fork Fork, upgradeTime time.Time) upgrade.Config {
+	c := upgrade.Config{
+		ApricotPhase4MinPChainHeight: 0,
```

Comment:
this is the zero value so why explicitly set it?

Reply:
We could remove this - I felt like it was a useful shout-out... But I'm fine deleting it.

Reply:
removed.